src/DB/exerciseData.json

```diff
@@ -10,7 +10,8 @@
     ],
     "videoLink": "https://youtu.be/ykJmrZ5v0Oo",
     "gh-name": "druvkotwani",
-    "image": "https://www.inspireusafoundation.org/wp-content/uploads/2023/01/inner-bicep-curl.gif"
+    "image": "https://www.inspireusafoundation.org/wp-content/uploads/2023/01/inner-bicep-curl.gif",
+    "tags":["Biceps"]
```

Comment:
I think it’s best to move all the tags to a separate file. We can import the tags here and use. This will ensure non duplication of tags. By this way we reuse, the string also.